templates/master/01-master-kubelet/_base/files/kubelet.yaml

```diff
@@ -26,11 +26,7 @@ contents:
       ephemeral-storage: 1Gi
     featureGates:
       APIPriorityAndFairness: true
-      LegacyNodeRoleBehavior: false
```

Comment:
would prefer not to pull in these changes for this PR as they are contained in https://github.com/openshift/machine-config-operator/pull/2949/files which should be merging soon.

Reply:
feel free to ignore bc the other pr will merge first :)

Reply:
Cool! Will rebase that in.